lib/galaxy/util/__init__.py

```diff
@@ -335,6 +335,17 @@ def shrink_string_by_size( value, size, join_by="..", left_larger=True, beginnin
     return value
 
 
+def parse_query_string( query_string ):
```

Comment:
Brilliant, thanks! I think that now that you've added `if '=' not in argument`, you can remove:
```
    if '=' not in query_string:
        return {}
```